### DISCUSSION TITLE
OSMAnd photos moving

### DISCUSSION THREAD

**Discussion by HikeAndMap** [osmandapp/OsmAnd#13542] [Q&A] [answered]
When I make the photo with the photo plugin.. and I see the location isn't 100% then I move it manually to the "right" spot - at least to be more accurate.

in the past I believe to remember JOSM then loads these images and shows where I moved the image to.

Now I'm loading these images in JOSM and it shows the original taken locations - not the locations where I moved the images to.

and jExifToolGUI v1.8.1 claims there's makernotes - warning - invalid metadata

below a copy of one of the many files that has the "warning makernote invalid metadata"

![4zexDuyI-- 1](https://user-images.githubusercontent.com/15773876/148859662-0f46448e-ccb2-4e7b-bc8b-029b6383732f.jpg)

So I'm going out of a limb here - that's where the GPS moved data is stored and neither exiftools not JOSM can read the data?

Anyway I can fix that?

**Comment by vshcherb** [comment, marked as answer]
Osmand stores location inside the name ! It stores in exif data too but it's not a supported way for some environments. 
Here is the algorithm how location is encoded inside name of a/v notes - https://wiki.openstreetmap.org/wiki/Shortlink#:~:text=A%20shortlink%20is%20a%20link,option%20on%20the%20Share%20tab.
